webrtc-stats.html

```diff
@@ -1996,6 +1996,23 @@ <h2>
                   level = -round(log10(audioLevel) * 20), with audioLevel 0.0 and values above 127
                   mapped to 127.
                 </p>
+                <p>
+                  The <a>audioLevel</a> represents the output audio
+                  level of the track; thus, if the track is sourced
+                  from an <a>RTCReceiver</a>, does no audio
+                  processing, has a constant level, and has
+                  a <a>volume</a> setting of 1.0, the audio level is
+                  expected to be the same as the audio level of the
+                  source SSRC, while if the <a>volume</a> setting is
+                  0.5, the audioLevel is expected to be half that
+                  value.
+                </p>
+                <p>
+                  The audioLevel is averaged over some small
+                  interval, using the algortihm described
```

Comment:
This "small interval" part is new; what was the assumption before? That it was just the last decoded/sent packet?

Reply:
The previous assumption was that it wasn't mentioned; I think the code averaged over a packet. This leaves it specified as "implementation defined", but the averaging has to use the sum-of-squares method rather than just averaging (averaging gives a completely bogus number).